content/200-concepts/150-data-platform/01-about-platform/100-platform-limits-and-status.mdx

```diff
@@ -0,0 +1,84 @@
+| --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- |
+| Maximum number of projects on the **Free** plan per account<br /><br /><br />**Note**<br /><br /> The number does not include projects on the **Early Adopter** plan. | 10             |
+| Maximum number of projects per account                                                                                                                                | 100            |
+| Maximum number of collaborators per project                                                                                                                           | 100            |
+| Maximum number of environments per project                                                                                                                            | 100            |
+| Maximum number of connection strings per environment                                                                                                                  | 100            |
+| Response timeout in Data Browser                                                                                                                                      | 60s            |
+| Response timeout in Query Console                                                                                                                                     | 60s            |
+| Maximum number of rows fetched in Data Browser or Query Console                                                                                                       | Not applicable |
+
+## Concurrency limits of the Prisma Data Proxy
+
+The concurrency limits of the Prisma Data Proxy define the maximum number of proxy instances that can run simultaneously for a single project. By default, only one proxy instance is started for a single project environment.
+
+More proxy instances start when:
+
+- you create and start using a new environment
+- the number of running proxy instances cannot handle the number of incoming queries
+- multiple Prisma Clients connect that use a different version of the Prisma Client
+- multiple Prisma Clients connect that use a different version of the Prisma schema
+
+| Concurrency limits                                                     | Number |
+| ---------------------------------------------------------------------- | ------ |
+| Maximum number of concurrent proxy instances on **Early Adopter** plan | 3      |
+| Maximum number of concurrent proxy instances on **Free** plan          | 100    |
+| Maximum number of concurrent proxy instances on **Scale** plan         | 100    |
+| Maximum number of concurrent proxy instances on **Enterprise** plan    | Custom |
+
+A single proxy instance can handle ~1200 RPS which translates to ~69 concurrent database connections.
```

Comment:
This is not generally true. I arrived at that number with an assumption that a query takes a certain amount of time, which is only an assumption that the user can make. Like, we don't know how complex their queries are.

Reply:
Thanks! It is now updated based on the offline discussion we had.